sklearn/linear_model/tests/test_ridge.py

```diff
@@ -1234,60 +1234,58 @@ def test_ridgecv_int_alphas():
     ridge.fit(X, y)
 
 
-def test_ridgecv_scalar_alphas():
+@pytest.mark.parametrize(
+    "params, err_type, err_msg",
+    [
+        ({"alphas": (1, -1, -100)}, ValueError, r"alphas\[1\] == -1, must be > 0.0"),
+        (
+            {"alphas": (-0.1, -1.0, -10.0)},
+            ValueError,
+            r"alphas\[0\] == -0.1, must be > 0.0",
+        ),
+        (
+            {"alphas": (1, 1.0, "1")},
+            TypeError,
+            r"alphas\[2\] must be an instance of <class 'numbers.Real'>, not <class"
+            r" 'str'>",
+        ),
+    ],
+)
+def test_ridgecv_scalar_alphas(params, err_type, err_msg):
+    """Check the parameters validation in RidgeCV."""
```

Comment:
Perfect. :+1: 

Probably, it is even possible to merge this test with the next one by parametrising on `[RidgeClassifierCV, RidgeCV]` using a `Estimator` parameter.

Reply:
Thanks! Done!